docs/sdk.md

```diff
@@ -0,0 +1,88 @@
+Calypso SDK
+===========
+
+Calypso SDK is an early stage tool that's goal is to build, visualize, test, and deliver extensions from a single source.
+
+## Using SDK CLI
+
+```bash
+npm run sdk -- --help
+```
+
+Note: It's also possible to run the SDK command "globally" by linking within the Calypso repository with [`npm link`](https://docs.npmjs.com/cli/link). After running this command you can replace all invocations of `npm run sdk --` in the examples below with `calypso-sdk` and may do so from any other directory in the filesystem:
+```
+calypso-sdk --help
+```
+
+## SDK modules
+
+SDK can be extended with modules to perform different tasks. Currently we have only one task; build Gutenberg extensions.
+
+### Gutenberg extensions
+
+SDK module to build [Gutenberg](https://wordpress.org/gutenberg/handbook/) extensions.
+
+See usage instructions:
+
+```
+npm run sdk -- gutenberg --help
+```
+or
+```
+npm run sdk:gutenberg -- --help
```

Comment:
Should we deprecate `sdk:gutenberg` and leave it undocumented?

Reply:
or just remove it? in the end I think we'll not want to create a new `bin` entry for every target

Reply:
Agreed & removed.